### DISCUSSION TITLE
Prisma for other languages

### DISCUSSION THREAD

**Discussion by tyteen4a03** [prisma/prisma#14325] [Q&A] [answered]
Are there any plans to bring Prisma to other languages? It would be a real winner if we can share a `prisma.schema` across languages.

**Comment by TasinIshmam** [comment, marked as answer]
Hi @tyteen4a03  

First-party support for other languages isn't planned anytime soon. This is something we might do in the future, but we can't really give you anything definitive right now. 

[Our roadmap](https://pris.ly/roadmap) lists the features that are planned right now.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
